### DISCUSSION TITLE
Conflict in styles using Material UI. right after building and serving

### DISCUSSION THREAD

**Discussion by MohammedSaberMohammed** [vercel/next.js#29967] [Help]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

v14.16.0

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next build, next export, next start

### Describe the Bug

Hello every one, 
I'm building an application and when I run `npm run dev` the application works fine like this 
<img width="1656" alt="Screen Shot 2021-10-16 at 1 59 11 PM" src="https://user-images.githubusercontent.com/19964514/137586468-acb10c82-06a4-4c03-9eb2-5171c2c5d2a7.png">

but when I serve from `npm run start` this is the result 
All my styles overrided by the MUI generated classes like `css-classId`

<img width="1656" alt="Screen Shot 2021-10-16 at 1 57 43 PM" src="https://user-images.githubusercontent.com/19964514/137586565-4e32d315-ed6a-4dae-842b-7d41f8aaf607.png">

### Expected Behavior
 
The app should be the same ( styles and functionalities )

### To Reproduce

npm run build
npm run export
npm run start
npm run dev

"@mui/material": "^5.0.0",
"styled-components": "^5.3.1"
"react": "^17.0.2",
"react-dom": "^17.0.2",

**Comment by jonasstams** [comment]
I'm having the same issue

**Reply by tgmarinho** [reply]
same behavior here with MUI v4 and Next 12
